packages/api/source/utils/index.ts

```diff
@@ -0,0 +1,228 @@
+const invitationTemplate = (token) => {
```

Comment:
Move this to a template file.

Reply:
I don't see the point of embedding HTML in JS, when we can separate HTML to a separate file completely.

Reply:
How will we paas variables then?